src/tree/assess/assessments/Assessment.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * An assessment that can be applied to a formatted text and its meta data.
+ *
+ * @memberOf module:tree/assess
+ *
+ * @abstract
+ */
+class Assessment {
+	/**
+	 * Creates a new assessment.
+	 *
+	 * @param {string} [name] An optional name.
```

Comment:
Yep, we need to add JSDocs to these properties inside of the constructor, unfortunately 😞 .

Reply:
I made `name` a required constructor argument.
This makes sense, since you should be able to retrieve an assessment by its name using the `TreeAssessor`, even when you give the assessment as a `TreeAssessor` constructor argument.